pytest/lib/account.py

```diff
@@ -62,9 +63,11 @@ def prep_tx(self):
         self.tx_timestamps.append(time.time())
         self.nonce += 1
 
-    def send_transfer_tx(self, dest_account_id, base_block_hash=None):
+    def send_transfer_tx(self,
+                         dest_account_id,
+                         transfer_amount=100,
+                         base_block_hash=None):
```

Comment:
Perhaps

    def send_transfer_tx(self,
                         dest_account_id,
                         *,
                         transfer_amount=100,
                         base_block_hash=None):

to make sure existing callers don’t pass base block hash as transfer
amount?  This will force using keyword arguments.

Reply:
> existing callers

Checked that none of the existing callers get affected by this change.